Source/BugsnagMetadata.m

```diff
@@ -45,13 +45,30 @@ - (id)initWithDictionary:(NSDictionary *)dict {
     if (self = [super init]) {
         // Ensure that the instantiating dictionary is mutable.
         // Saves checks later.
-        self.dictionary = [dict mutableCopy];
+        self.dictionary = [self sanitizeDictionary:[dict mutableCopy]];
```

Comment:
Suggest moving the mutable copy inside sanitizeDictionary to better contain the functionalty.